R/expect_identical_linter.R

```diff
@@ -1,6 +1,6 @@
-#' Require usage of expect_identical(x, y) where appropriate
+#' Require usage of `expect_identical(x, y)` where appropriate
 #'
-#' At Google, [testthat::expect_identical()] should be the default/go-to function for
```

Comment:
"At Google" was intentional here -- I can't find where it was mentioned, but Hadley mentioned this is not really true for testthat in general. Maybe a more general way to phrase this first sentence is:

This linter enforces the usage of [testthat::expect_identical()] as the default expectation for comparisons in a testthat suite.

Reply:
Ah, I see. 

Okay, I have changed the first sentence to the one you have suggested.